pom.xml

```diff
@@ -114,6 +114,18 @@
             <artifactId>aether-spi</artifactId>
             <version>1.0.0.v20140518</version>
         </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>job-dsl</artifactId>
+            <version>1.35</version>
+            <optional>true</optional>
+        </dependency>
+        <dependency>
+            <groupId>org.jenkins-ci.plugins</groupId>
+            <artifactId>matrix-project</artifactId>
```

Comment:
Can't see why we'd need that kind of dependency. `matrix` is only mentioned in this line.